views/home/patientHome.php

```diff
@@ -36,5 +36,179 @@
+          $sql = "SELECT morning_med, afternoon_med, night_med, breakfast, lunch, dinner
+          FROM schedules
+          WHERE user_id = '$id' AND the_date = '$date'";
+
+          $result = mysqli_query($link, $sql);
+          if ($result) $schedule = $result->fetch_assoc();
+
+          #get the group number that the user is in
+          $sql = "SELECT patient_group FROM patients_info WHERE user_id = '$id'";
+          $result = mysqli_query($link, $sql);
+          if ($result) {
+            $row = $result->fetch_assoc();
+
+            $group = $row['patient_group'];
+
+            #get the appropriate caregiver from the roster if there is a roster
+            $sql = "SELECT doctor, caretaker_1, caretaker_2, caretaker_3, caretaker_4
+            FROM rosters
+            WHERE the_date = '$date'";
+
+            $result = mysqli_query($link, $sql);
+
+            if ($result) {
+
+              $row = $result->fetch_assoc();
+
+              if($row) {
+
+                if ($group == 1) {
+                  $caretaker = $row['caretaker_1'];
+                }
+                elseif ($group == 2) {
+                  $caretaker = $row['caretaker_2'];
+                }
+                elseif ($group == 3) {
+                  $caretaker = $row['caretaker_3'];
+                }
+                elseif ($group == 4) {
+                  $caretaker = $row['caretaker_4'];
+                }
+
+                #save the doctor id for later
+                $doctor = $row['doctor'];
+
+                #get the caretaker's name
+                $sql = "SELECT Fname, Lname
+                FROM users
+                WHERE id = $caretaker";
+
+                $result = mysqli_query($link, $sql);
+                if ($result) {
+
+                  $row = $result->fetch_assoc();
+
+                  if ($row) $caretaker_name = $row['Fname'] . ' ' . $row['Lname'];
+                }
+                else {
+                  echo "ERROR: Unable to establish connection to database";
+                }
+
+                #get the doctor's name using the id from earlier
+                $sql = "SELECT Fname, Lname
+                FROM users
+                WHERE id = $doctor";
+
+                $result = mysqli_query($link, $sql);
+                if ($result) {
+
+                  $row = $result->fetch_assoc();
+
+                  if ($row) $doctor_name = $row['Fname'] . ' ' . $row['Lname'];
+                }
+                else {
+                  echo "ERROR: Unable to establish connection to database";
+                }
+              }
+            }
+            else {
+              echo "ERROR: Unable to establish connection to database";
+            }
+          }
+          else {
+            echo "ERROR: Unable to establish connection to database";
+          }
+
+          #check if there is an appointment today
+          $sql = "SELECT * FROM appointments
+          WHERE the_date = '$date' AND patient_id = '$id'";
```

Comment:
In the database, you changed schedules from the_date to day I think it would be consistent to do the same where applicable in other tables.